glimr/search.py

```diff
@@ -324,16 +327,18 @@ def trainable(config):
             as well as functions for data loading and model building.
         """
 
-        # create the model from the config
-        model, losses, loss_weights, metrics = config["builder"](config)
+        strategy = tf.distribute.MultiWorkerMirroredStrategy()
+        with strategy.scope():
```

Comment:
Yes, with MultiWorkerMirroredStrategy we can train a model (trial) on different GPUs. Even if we set ScalingConfig there since we are using MultiWorkerMirroredStrategy the data (as a result of that operations in a model) will be distributed across all the GPUs.